tests/layer_tests/tensorflow_tests/test_tf_Conv3DBackprop.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+import logging
+
+# Testing operation Conv3DBackpropInputV2
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/Conv3DBackpropInputV2
+
+class TestConv3DBackprop(CommonTFLayerTest):
+    # Overrides regular input generator to reduce range
+    def _prepare_input(self, inputs_dict):
+        for input in inputs_dict.keys():
+            inputs_dict[input] = np.random.randint(-32, 32, inputs_dict[input]).astype(np.float32)
+        return inputs_dict
+
+    # input_shape - should be an array, shape of input tensor in format [batch, depth, height, width, channels]
+    # input_filter - should be an array, defines a filter
+    # input_strides - should be an array, defines strides of a sliding window to use
+    # input_padding - should be a string, defines padding algorithm
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
+    def create_conv3dbackprop_placeholder_const_net(self, input_shape, input_filter, out_backprop, input_strides, input_padding, dilations, ir_version, use_new_frontend):
+        """
+            Tensorflow net                                 IR net
+
+            Placeholder->Conv3DBackpropInputV2    =>       Placeholder->Transpose->Convolution->Transpose
```

Comment:
nit: need to replace `Convolution` to `ConvolutionBackpropData`

Reply:
done